api/tests/test_application_round_api.py

```diff
@@ -79,17 +86,134 @@ def test_application_round_when_basket_orders_dont_overlap(
 
 @pytest.mark.django_db
 def test_update_application_round(
-    user, user_api_client, application_round, valid_application_round_data
+    user_api_client,
+    service_sector_admin_api_client,
+    application_round,
+    valid_application_round_data,
 ):
     data = {**valid_application_round_data, "status": ApplicationRoundStatus.PUBLISHED}
+
+    # Normal user not allowed to edit application round
     response = user_api_client.put(
         reverse("application_round-detail", kwargs={"pk": application_round.id}),
         data=data,
         format="json",
     )
+    assert response.status_code == 403
+
+    # Service Sector admin can edit application round
+    response = service_sector_admin_api_client.put(
+        reverse("application_round-detail", kwargs={"pk": application_round.id}),
+        data=data,
+        format="json",
+    )
     assert response.status_code == 200
 
     application_round.refresh_from_db()
 
     assert application_round.status == ApplicationRoundStatus.PUBLISHED
     assert application_round.name == valid_application_round_data["name"]
+
+
+def test_application_round_permissions(
+    application_round,
+    valid_application_round_data,
+    user_api_client,
+    user_2_api_client,
+    general_admin_api_client,
+    service_sector_admin_api_client,
+    service_sector_2_admin_api_client,
+    service_sector_application_manager_api_client,
+    unit_admin_api_client,
+):
+    # Normal user can't create application rounds
+    response = user_api_client.post(
+        reverse("application_round-list"),
+        data=valid_application_round_data,
+        format="json",
+    )
+    assert response.status_code == 403
```

Comment:
A bit long and multi assert test here too.

Reply:
Changed to separate tests